server.go

```diff
@@ -27,16 +27,21 @@ import (
 
 	"github.com/RedHatInsights/insights-content-service/conf"
 	"github.com/RedHatInsights/insights-content-service/groups"
+	"github.com/RedHatInsights/insights-content-service/server"
 )
 
 const (
 	// ExitStatusOK means that the tool finished with success
 	ExitStatusOK = iota
 
+	// ExitStatusServerError is returned in case of any REST API server-related error
+	ExitStatusServerError
```

Comment:
it is not strictly needed, but IMHO it might be better to "group" all enum values in one separated `const` block

Reply:
Do you mean separating the "ExitStatus" consts from the rest, like the `defaultConfigFilename`?

Reply:
yes